bzt/modules/console.py

```diff
@@ -114,8 +115,8 @@ def prepare(self):
         if isinstance(self.engine.aggregator, ResultsProvider):
             self.engine.aggregator.add_listener(self)
 
-        self.disabled = self.settings.get("disable", False)
-        if self.disabled:
+        if (not self.is_tty) or (self.settings.get("disable", False)):
```

Comment:
There is a case when user wants to still enable console on non-tty. Better have "true/false/auto" for "disable"